crates/bevy_feathers/src/controls/slider.rs

```diff
@@ -0,0 +1,197 @@
+            on_change: None,
+            // overrides: (),
+        }
+    }
+}
+
+#[derive(Component, Default, Clone)]
+#[require(CoreSlider)]
+struct SliderStyle;
+
+/// Marker for the text
+#[derive(Component, Default, Clone)]
+struct SliderValueText;
+
+/// Spawn a new slider widget.
+pub fn slider(props: SliderProps) -> impl Bundle {
+    (
+        Node {
+            height: size::ROW_HEIGHT,
+            justify_content: JustifyContent::Center,
+            align_items: AlignItems::Center,
+            padding: UiRect::axes(Val::Px(8.0), Val::Px(0.)),
+            flex_grow: 1.0,
+            ..Default::default()
+        },
+        CoreSlider {
+            on_change: props.on_change,
+            track_click: TrackClick::Drag,
+        },
+        SliderStyle,
+        SliderValue(props.value),
+        SliderRange::new(props.min, props.max),
+        CursorIcon::System(bevy_window::SystemCursorIcon::EwResize),
+        TabIndex(0),
+        RoundedCorners::All.to_border_radius(6.0),
+        // Use a gradient to draw the moving bar
+        BackgroundGradient(vec![Gradient::Linear(LinearGradient {
+            angle: PI * 0.5,
+            stops: vec![
+                ColorStop::new(Color::NONE, Val::Percent(0.)),
+                ColorStop::new(Color::NONE, Val::Percent(50.)),
+                ColorStop::new(Color::NONE, Val::Percent(50.)),
+                ColorStop::new(Color::NONE, Val::Percent(100.)),
+            ],
+        })]),
+        // props.overrides,
+        children![(
+            // Text container
+            Node {
+                display: Display::Flex,
+                flex_direction: FlexDirection::Row,
+                align_items: AlignItems::Center,
+                justify_content: JustifyContent::Center,
+                ..Default::default()
+            },
+            ThemeFontColor(theme::tokens::SLIDER_TEXT),
+            InheritableFont {
+                font: HandleOrPath::Path(fonts::MONO.to_owned()),
+                font_size: 12.0,
+            },
+            children![(Text::new("10.0"), UseTheme, SliderValueText,)],
+        )],
+    )
+}
+
+fn update_slider_colors(
+    mut q_sliders: Query<
+        (Entity, Has<InteractionDisabled>, &mut BackgroundGradient),
+        (With<SliderStyle>, Or<(Spawned, Added<InteractionDisabled>)>),
+    >,
+    theme: Res<UiTheme>,
+) {
+    for (_slider_ent, disabled, mut gradient) in q_sliders.iter_mut() {
+        let bar_color = theme.color(match disabled {
+            true => SLIDER_BAR_DISABLED,
+            false => SLIDER_BAR,
+        });
+        let bg_color = theme.color(SLIDER_BG);
+        if let [Gradient::Linear(linear_gradient)] = &mut gradient.0[..] {
+            linear_gradient.stops[0].color = bar_color;
+            linear_gradient.stops[1].color = bar_color;
+            linear_gradient.stops[2].color = bg_color;
+            linear_gradient.stops[3].color = bg_color;
+        }
+    }
+}
+
+fn update_slider_colors_remove() {}
```

Comment:
I don't understand why this exists.

Reply:
Because I forgot to write this function :)